cloud_sql_proxy/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM scratch
```

Comment:
The cloudsql proxy is a statically-built container. It doesn't need anything else (other than the root x509 certs for TLS connections, but those already reside on the VM are are mapped into the container).